app/models/wds-on-site-notifications/wds-on-site-notification.js

```diff
@@ -77,9 +78,29 @@ export default EmberObject.extend({
 				return notificationTypes.articleCommentAtMention;
 			case 'article-comment-reply-at-mention-notification':
 				return notificationTypes.articleCommentReplyAtMention;
+			case 'message-wall-post-notification':
+				return notificationTypes.messageWallPost;
+			case 'message-wall-reply-notification':
+				return notificationTypes.messageWallReply;
+			case 'wall-post-removed-notification':
+				return notificationTypes.messageWallPostRemoved;
 		}
 	},
 
+	getMetadata(notification) {
+		let metadata =
+			typeof notification.metadata === 'undefined' || notification.metadata === null ? null : notification.metadata;
```

Comment:
Why not `let metadata = notification.metadata || null` or better (if we have proper settings) `let metadata = notification.metadata ?? null`

Reply:
Yep. that works too. making the change.